Documentation/policy/language.rst

```diff
@@ -274,11 +274,11 @@ CIDR policies are used to define policies to and from endpoints which are not
 managed by Cilium and thus do not have labels associated with them. These are
 typically external services, VMs or metal machines running in particular
 subnets. CIDR policy can also be used to limit access to external services, for
-example to limit external access to a particular IP range.
+example to limit external access to a particular IP range. These policies never
+apply to traffic that comes from Cilium endpoints and is destined to Cilium
```

Comment:
Do you want to also clarify,  if a specific CIDR range contains POD IPs, those will be ignored?